### PR TITLE
chore!: drop support for Python 3.9, add support for Python 3.11

### PR DESCRIPTION
**Pull request by jenstroeger**

### CODE REVIEW

**Review by behnazh — APPROVED** [review]

File: .github/workflows/_build.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ jobs:
         # It is recommended to pin a Runner version specifically:
         # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python: ['3.9', '3.10']
+        python: ['3.10', '3.11']
```
Comment (behnazh) [inline .github/workflows/_build.yaml:47]: What's the point of supporting two Python versions only? Wouldn't it make sense to use packages like [cibuildwheel](https://cibuildwheel.readthedocs.io/en/stable/) to manage the Python version and OS?

Comment (jenstroeger) [reply .github/workflows/_build.yaml:47]: I’m not familiar with that package, but it looks interesting. Let’s move your suggestion to a separate issue?

**Review by behnazh — APPROVED** [review]

File: .github/workflows/_build.yaml (hunk 1 above)
Comment (behnazh) [reply .github/workflows/_build.yaml:47]: > I’m not familiar with that package, but it looks interesting. Let’s move your suggestion to a separate issue?

Sounds good, thanks.

Comment (jenstroeger) [reply .github/workflows/_build.yaml:47]: Issue https://github.com/jenstroeger/python-package-template/issues/451.

**Comment by jenstroeger** [comment]
@behnazh I’m unsure about the breaking change `chore!:` here 🤔 What do you think?

**Comment by behnazh** [comment]
> @behnazh I’m unsure about the breaking change `chore!:` here thinking What do you think?

It's a tricky one because this may or may not be a breaking change depending on the downstream repositories. I would slightly prefer to make it a breaking change to communicate the potential side effects?

**Comment by jenstroeger** [comment]
> It's a tricky one because this may or may not be a breaking change depending on the downstream repositories. I would slightly prefer to make it a breaking change to communicate the potential side effects?

Ok then let’s keep the PR title as-is 👍🏼

**Pull request merged**